CONTRIBUTING.md

```diff
@@ -169,6 +169,21 @@ To start n8n with tunnel:
 ./packages/cli/bin/n8n start --tunnel
 ```
 
+## First run
+
+The first time you run n8n locally you will be prompted to create an owner user.
```

Comment:
Is this information really needed? it is the same as normally running n8n so this shouldn't be a surprise.

Reply:
It's definitely not a suprise during first run. It still adds some context for users that do need to reset the credentials (e.g. people that tried running n8n locally once or twice and return back to it a couple of weeks later). So, I'd say let's keep it but I am perfectly fine with removing it.